packages/common/src/polyfills/version/release-version.ts

```diff
@@ -0,0 +1,39 @@
+// default return values
+const OLD_DEV = [-1 ,0 ,0];
+const DEV = [0, 0, 0]
+
+/**
+ * get the console release version
+ * 
+ * @returns a tupple with the console release version, on none release it will return 0, 0, 0,
+ *          on old development server, return -1, 0, 0
+ */
+const getReleaseVersion = () => {
+    const releaseVersion = window['SERVER_FLAGS']?.releaseVersion;
+    const consoleVersion = window['SERVER_FLAGS']?.consoleVersion;
+
+    if (!releaseVersion || typeof releaseVersion !== 'string' || releaseVersion === '') {
+        // if we don't have releaseVersion try consoleVersion
+        if (consoleVersion && typeof consoleVersion === 'string' && consoleVersion.startsWith('v6.0.6-20')) {
+            return OLD_DEV;
+        }
+
+        return DEV;
+    }
+    const [major, minor, bugfix] = releaseVersion.split('.').map((v) => parseInt(v, 10));
+
+    return [major, minor, bugfix]
+}
+
+/**
+ * Specific version check based on getReleaseVersion
+ * 
+ * @returns true if version is 4.11 or lower
+ */
+const isConsoleVersion_4_11 = () => {
+    const [major, minor] = getReleaseVersion();
```

Comment:
Just in case I would wrap the  utility with `try-catch`

Reply:
why ?

Reply:
parsing is always fragile.  We had problems with that in oVirt. Basically semantic versioning allows non-numeric strings like `1.0.0-beta`

Reply:
yes, but the fail check is done on `getReleaseVersion` that is the method that do the parsing, and it is the one that will fall back to `DEV` when parsing fails